src/rdf4cpp/datatypes/xsd/time/Month.cpp

```diff
@@ -25,11 +25,15 @@ capabilities::Default<xsd_gMonth>::cpp_type capabilities::Default<xsd_gMonth>::f
 
 template<>
 bool capabilities::Default<xsd_gMonth>::serialize_canonical_string(cpp_type const &value, writer::BufWriterParts writer) noexcept {
-    auto str = std::format("--{:%m}", value.first);
-    if (value.second.has_value())
-        str += value.second->to_canonical_string();
-
-    return writer::write_str(str, writer);
+    //assumes month is in [1,12]
```

Comment:
Can month be larger than this interval?

Reply:
std::chrono::month is [0,255], but from_string checks, that it is in [1,12]

Reply:
I would just add +1 for safety then. I don't think this can happen but better safe than sorry